batch/environment.yml

```diff
@@ -11,3 +11,13 @@ dependencies:
   - pylint
   - pytest
   - requests
+  - aiohttp
+  - aiodns
+  - cchardet
+  - aiohttp_jinja2
+  - jinja2
+  - uvloop>=0.12
```

Comment:
Do we need the version resolution? The current package is 0.12.1 right? I’m actually not sure when > resolution is needed, unless we can’t rely on the package repo to point version to latest?

Reply:
See here: 
https://github.com/MagicStack/uvloop#using-uvloop

Syntax for using uvloop changed with that version.

Reply:
Right, I understand why you want >= 0.12, I just meant that by default, if we write 
`
- uvloop
` 

(no version) are there circumstances in which a version < current will be fetched? I'm more used to constraining versions to be something smaller than current, such as ^0.12.3 or <0.19.